package.json

```diff
@@ -201,7 +201,7 @@
     "yosay": "^1.2.1"
   },
   "devDependencies": {
-    "@ciscospark/eslint-config": "^1.0.0",
+    "@ciscospark/eslint-config": "^2.0.0",
```

Comment:
I may have misunderstood then; the problem with that is that I could not verify that the branch worked correctly and would have failed all CircleCI and ESLint tests unless @ciscospark/ciscospark-eslint-config was published to npm as @ciscospark/eslint-config. I apologize if this was something i shouldn't have done here. Do we need to roll back, and if so what steps need to be taken?